ethereum/contracts/pyth/PythInternalStructs.sol

```diff
@@ -62,17 +33,4 @@ contract PythInternalStructs {
         uint16 chainId;
         bytes32 emitterAddress;
     }
-
-    /* PriceAttestationStatus represents the availability status of a price feed passed down in attestation.
-        UNKNOWN: The price feed is not currently updating for an unknown reason.
-        TRADING: The price feed is updating as expected.
-        HALTED: The price feed is not currently updating because trading in the product has been halted.
-        AUCTION: The price feed is not currently updating because an auction is setting the price.
-    */
-    enum PriceAttestationStatus {
```

Comment:
i take it the enum costs extra gas ? it's a lot nicer to have the enum than a number so would be nice to keep this if possible.

Reply:
Probably there is a gas cost, maybe not much. But I rather keep it out as we soon will remove status and previous fields from price attestation (it will also save gas + makes logic simpler)